src/lib/shopping-fpti/event-handlers.js

```diff
@@ -32,10 +37,11 @@ export function eventSinfoBuilderInit(config : Config) : Object {
 }
 
 export function customEventMappingInit(config : Config) : Object {
-  function enrichPageViewEvent() : Object {
+  function enrichPageViewEvent(payload : Object) : Object {
+    const disableStoreCash = findConfigurationAttribute(config, payload, 'disable_storecash') || 'false';
     const containerSummary = config.containerSummary || {};
     const offerId = containerSummary.programId;
-    if (offerId) {
+    if ((disableStoreCash === 'false') && offerId) {
```

Comment:
Is there a reason we need to use a string for `disable_storecash` property?

Reply:
Yes, we need to be able to have an option to disable store cash on any page